cleverhans/experimental/certification/optimization.py

```diff
@@ -241,6 +240,13 @@ def run_one_step(self, eig_init_vec_val, eig_num_iter_val,
     Returns:
      found_cert: True is negative certificate is found, False otherwise
     """
+      
```

Comment:
This causes a crash when the optimizer tries to change the graph inside the loop. I added it to automatically find places where we add nodes when we shouldn't. We can take it back out when we're done with the refactor.